cmd/virt-launcher/qemu-kube

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
```

Comment:
You will have to rebase, once we have merged #195 because of the cgroup changes ...